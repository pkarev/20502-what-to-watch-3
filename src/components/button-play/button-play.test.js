import React from 'react';
import renderer from 'react-test-renderer';
import ButtonPlay from './button-play.jsx';

it(`Render ButtonPlay`, () => {
  const tree = renderer
    .create(<ButtonPlay onClick={() => {}}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
