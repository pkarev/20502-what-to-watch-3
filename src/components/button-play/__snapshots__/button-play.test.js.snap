// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render ButtonPlay 1`] = `
<button
  className="btn btn--play movie-card__button"
  onClick={[Function]}
  type="button"
>
  <svg
    height="19"
    viewBox="0 0 19 19"
    width="19"
  >
    <use
      xlinkHref="#play-s"
    />
  </svg>
  <span>
    Play
  </span>
</button>
`;
